doc/client_setup.md

```diff
@@ -0,0 +1,199 @@
+## Setting up a new C2 client
+
+This document describes the details of setting up a new C2 client.
+
+### Overview
+
+A *client* represents a group of Users within the C2 system that share
+a data model. Existing examples include **GSA18F** and **NCR**.
+
+Within a *client* users may be assigned different roles. See the [Authorization](roles.md)
+document for details on roles.
+
+A *client* namespace is assigned to each User via a `client_slug` model value. The
+convention is that the `client_slug` is a downcased string matching the client model
+namespace. For example `ncr` for **NCR**.
+
+A User without an assigned `client_slug` may not create, edit, view or search Proposals,
```

Comment:
@jessieay you might be seeing something different than "normal" users because you are an admin. These tests demonstrate that "normal" users are limited based on `client_slug`.

https://github.com/18F/C2/blob/master/spec/features/client_slug_authz_spec.rb